pkgs/development/compilers/dmd/default.nix

```diff
@@ -117,11 +117,17 @@ stdenv.mkDerivation rec {
 
   checkPhase = ''
     cd dmd
-    make -j$NIX_BUILD_CORES -C test -f Makefile PIC=1 CC=$CXX DMD=${pathToDmd} BUILD=release SHELL=$SHELL
+    NIX_ENFORCE_PURITY= \
```

Comment:
is it possible to just to an `unset` or an `export`?

Reply:
I thought it might be beneficial to keep the value of `NIX_ENFORCE_PURITY` unchanged for the succeeding phases.

Reply:
I guess that's reasonable.